src/libecalc/core/graph_result.py

```diff
@@ -724,50 +850,124 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
+                        consumer_result.component_result.energy_usage, regularity=regularity
+                    ).to_calendar_day()
+                    if consumer_result.component_result.energy_usage.unit == Unit.STANDARD_CUBIC_METER_PER_DAY
+                    else TimeSeriesRate.from_timeseries_stream_day_rate(
+                        consumer_result.component_result.energy_usage, regularity=regularity
+                    ).to_stream_day(),
+                },
             )
+            if isinstance(obj, GeneratorSetResult):
+                obj.power_capacity_margin = TimeSeriesRate.from_timeseries_stream_day_rate(
+                    consumer_result.component_result.power_capacity_margin, regularity=regularity
+                )
+            elif isinstance(obj, PumpResult):
+                obj.inlet_liquid_rate_m3_per_day = TimeSeriesRate.from_timeseries_stream_day_rate(
+                    consumer_result.component_result.inlet_liquid_rate_m3_per_day, regularity=regularity
+                )
+                obj.inlet_pressure_bar = consumer_result.component_result.inlet_pressure_bar
+                obj.outlet_pressure_bar = consumer_result.component_result.outlet_pressure_bar
+                obj.operational_head = consumer_result.component_result.operational_head
+
+            elif isinstance(obj, CompressorResult):
+                obj.recirculation_loss = TimeSeriesRate.from_timeseries_stream_day_rate(
+                    consumer_result.component_result.recirculation_loss, regularity=regularity
+                )
+                obj.rate_exceeds_maximum = consumer_result.component_result.rate_exceeds_maximum
+                obj.outlet_pressure_before_choking = consumer_result.component_result.outlet_pressure_before_choking
+            elif isinstance(obj, AssetResult):
+                obj.hydrocarbon_export_rate = TimeSeriesRate.from_timeseries_stream_day_rate(
+                    consumer_result.component_result.hydrocarbon_export_rate, regularity=regularity
+                )
+                obj.emission_intensities = consumer_result.component_result.emission_intensities
+            elif isinstance(obj, InstallationResult):
+                obj.regularity = consumer_result.component_result.regularity
+
+            sub_components.append(obj)
+            # TODO: Is this skipped?
+            # )
 
         for installation in asset.installations:
+            regularity = regularities[installation.id]  # Already evaluated regularities
             for direct_emitter in installation.direct_emitters:
                 energy_usage = TimeSeriesRate(
                     timesteps=self.variables_map.time_vector,
                     values=[0.0] * self.variables_map.length,
-                    regularity=[1] * self.variables_map.length,
+                    regularity=[1] * self.variables_map.length,  # TODO: Correct? Dummy? Use calendar rate?
```

Comment:
I think dummy, I'm assuming regularity was unused for DirectEmitters. It seems to be a feature that is not used anyway, so we probably won't get much feedback if we do something else...